### PR TITLE
Hide showtoast in mobile devices

### PR DESCRIPTION
**Pull request by MaianaB**
**Feature/Bug description:** Some messages are unnecessary to show in mobile.

**Solution:** Show only error messages

**TODO/FIXME:** n/a

### CODE REVIEW

File: frontend/event/eventDetailsDirective.js [hunk 1]
```diff
@@ -163,7 +163,7 @@
         eventCtrl.copyLink = function copyLink() {
             var url = Utils.generateLink(`/event/${eventCtrl.event.key}/details`);
             ngClipboard.toClipboard(url);
-            MessageService.showToast("O link foi copiado");
+            MessageService.showInfoToast("O link foi copiado");
```
Comment (JuliePessoa) [inline frontend/event/eventDetailsDirective.js]: should show

File: frontend/institution/institutionCardDirective.js [hunk 2]
```diff
@@ -28,7 +28,7 @@
         institutionCardCtrl.copyLink = function copyLink(){
             var url = Utils.generateLink(URL_INSTITUTION + institutionCardCtrl.institution.key + "/home");
             ngClipboard.toClipboard(url);
-            MessageService.showToast("O link foi copiado");
+            MessageService.showInfoToast("O link foi copiado");
```
Comment (JuliePessoa) [inline frontend/institution/institutionCardDirective.js]: should show

File: frontend/institution/transferAdminController.js [hunk 3]
```diff
@@ -53,13 +53,13 @@
...
-                        MessageService.showToast("Convite enviado com sucesso!");
+                        MessageService.showInfoToast("Convite enviado com sucesso!");
                     });
                 } else {
-                    MessageService.showToast('Você já é administrador da instituição, selecione outro membro!');
+                    MessageService.showErrorToast('Você já é administrador da instituição, selecione outro membro!');
                 }
             } else {
-                MessageService.showToast('Selecione um memebro!');
+                MessageService.showErrorToast('Selecione um memebro!');
```
Comment (pedroespindula) [inline frontend/institution/transferAdminController.js]: Typo: "Selecione um membro"

File: frontend/post/postPageController.js [hunk 4]
```diff
@@ -28,7 +28,7 @@
         postCtrl.copyLink = function copyLink() {
             var url = Utils.generateLink('/post/' + postCtrl.post.key);
             ngClipboard.toClipboard(url);
-            MessageService.showToast("O link foi copiado");
+            MessageService.showInfoToast("O link foi copiado");
```
Comment (JuliePessoa) [inline frontend/post/postPageController.js]: should show

**Review by JuliePessoa — CHANGES_REQUESTED** [review]

File: frontend/test/specs/institution/registeredInstitutionsControllerSpec.js [hunk 5]
```diff
@@ -66,13 +67,13 @@ describe('registeredInstitutionController test', () => {
...
 
-            spyOn(msgService, 'showToast');
+            spyOn(msgService, 'showInfoToast');
 
             spyOn(authService, 'save');
 
             regInstCtrl.follow(regInstCtrl.institution.key);
 
-            expect(msgService.showToast).toHaveBeenCalled();
+            expect(msgService.showInfoToast).toHaveBeenCalledWith('Seguindo institution');
```
Comment (pedroespindula) [inline frontend/test/specs/institution/registeredInstitutionsControllerSpec.js:76]: The message that is used is right?

**Review by pedroespindula — CHANGES_REQUESTED** [review]

File: frontend/test/specs/institution/registeredInstitutionsControllerSpec.js (hunk 5 above)
Comment (MaianaB) [reply frontend/test/specs/institution/registeredInstitutionsControllerSpec.js:76]: Yes.

**Review by JuliePessoa — APPROVED** [review]

File: feature-toggles/test/specs/utils/messageServiceSpec.js [hunk 6]
```diff
@@ -11,11 +11,34 @@
...
 
+        describe("showErrorToast", function() {
             it('should call mdToast.show', function() {
+                window.screen = { width: 2000 };
+                spyOn(mdToast, 'show').and.callThrough();
+                service.showErrorToast("");
+                expect(mdToast.show).toHaveBeenCalled();
+            });
+
+            it('should not call mdToast.show', function() {
```
Comment (Luiz-FS) [inline feature-toggles/test/specs/utils/messageServiceSpec.js]: The description does not match the test

File: frontend/test/specs/utils/messageServiceSpec.js [hunk 7]
```diff
@@ -11,11 +11,34 @@
...
 
+        describe("showErrorToast", function() {
             it('should call mdToast.show', function() {
+                window.screen = { width: 2000 };
+                spyOn(mdToast, 'show').and.callThrough();
+                service.showErrorToast("");
+                expect(mdToast.show).toHaveBeenCalled();
+            });
+
+            it('should not call mdToast.show', function() {
```
Comment (Luiz-FS) [inline frontend/test/specs/utils/messageServiceSpec.js]: The description does not match the test

**Review by Luiz-FS — CHANGES_REQUESTED** [review]

**Review by pedroespindula — APPROVED** [review]

**Review by Luiz-FS — APPROVED** [review]

**Pull request merged**